pydantic/main.py

```diff
@@ -342,15 +359,13 @@ def copy(
         """
         if include is None and exclude is None and update is None:
             # skip constructing values if no arguments are passed
-            v = self.__values__
+            v = self.__values__.copy()
```

Comment:
do we need this copy?

Reply:
No, not if we remove the explicit add of `__field_set__` below.